docs/fundamentals/code-analysis/quality-rules/ca1000.md

```diff
@@ -59,15 +59,15 @@ To fix a violation of this rule, remove the static member or change it to an ins
 
 Do not suppress a warning from this rule. Providing generics in a syntax that is easy to understand and use reduces the time that is required to learn and increases the adoption rate of new libraries.
 
-## Configurability
+## Configure code to analyze
 
-You can configure which parts of your codebase to run this rule on, based on their accessibility. For example, to specify that the rule should run only against the non-public API surface, add the following key-value pair to an *.editorconfig* file in your project:
+You can configure this option for just this rule, for all rules, or for all rules in this category (Design). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
```

Comment:
```suggestion
You can configure these options for just this rule, for all rules, or for all rules in this category (Design). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
```

Change to plural to match line 66. Also, can you move this paragraph to line 70 (i.e. after index).

Reply:
I still think this paragraph would make more sense after the options are listed (i.e. line 70).

Reply:
My problem with that is that with markdown formatting it looks like the paragraph belongs to the last option and not to all options but I will let you have the final word here.

Reply:
Regarding the first suggestion, there is only one option for rule `CA1000` so `this option` makes more sense to me.

Reply:
Currently the CA1000 article looks like this (in the preview).

![image](https://user-images.githubusercontent.com/24882762/102130223-7003fc00-3e05-11eb-8044-d285814a13f7.png)

The important information should come first, and the important information is what the option(s) do. The info about configuration scope is secondary and should come afterwards.

Also, notice the pluralization discrepancy:

- You can configure this _option_...
- Use the following _options_...

Reply:
Oh you mean put the `Use the following` section before the `You can configure`? If so yes agreed.